ntfysh_client/SettingsDialog.cs

```diff
@@ -37,5 +73,20 @@ private void cancelButton_Click(object sender, EventArgs e)
         {
             DialogResult = DialogResult.Cancel;
         }
+
+        private void setNotificationsUIElements()
+        {
+            groupCustomNotificationSettings.Enabled = useCustomTrayNotifications.Checked;
+            timeoutLabel.Text = useCustomTrayNotifications.Checked ? _customNotificationsTimeout : _windowsNotificationsTimeout;
+            timeout.Minimum = useCustomTrayNotifications.Checked ? -1 : 0;
+        }
+
+        private void useCustomTrayNotifications_CheckedChanged(object sender, EventArgs e)
```

Comment:
Nitpick: Naming convention is capitalised first letter

Reply:
Changed